### ISSUE TITLE
Scheduler - horizontal and vertical group orientations

### ISSUE THREAD

**Issue by dxbykov** [DevExpress/DevExtreme#3712]
# The Problem

When horizontal scrolling and [grouping by resources](https://js.devexpress.com/Documentation/18_1/Guide/Widgets/Scheduler/Resources/Group_Appointments_by_Resources/) are enabled, it’s convenient to have static group titles on the left side. Previously, it was only possible to show groups horizontally in the top header. You can see the problem in action switching to the ‘Horizontal Grouping’ view in the [live demo](https://codepen.io/dxbykov/pen/MGgeaP).

# The Solution

Starting with 18.1, it’s possible to set the both `vertical` and `horizontal` orientation for the scheduler groups. We added the new [groupOrientation](https://js.devexpress.com/Documentation/18_1/ApiReference/UI_Widgets/dxScheduler/Configuration/views/#groupOrientation) option on a view configuration level. You can use it as follows:

```js
{
        views: [{
            ...
            groupOrientation: "vertical"
        }],
        ...
}

```

# Try It And Give Us Feedback 

## Live Sandbox

[jQuery](https://codepen.io/dxbykov/pen/MGgeaP) 
[Angular](https://embed.plnkr.co/pjc4XhpZintHPMhiPWDY/)

![scheduler_vertical_grouping](https://user-images.githubusercontent.com/8307562/38887494-ab5ccc14-4281-11e8-829a-228b23a2818e.png)

## Installation

Follow the installation guide from the [release page](https://github.com/DevExpress/DevExtreme/releases) and leave us feedback on this topic.

**Comment by wgutierrezr** [comment]
In the live demo, I tried to move an appointment to another day. When dropping the appointment to a particular day and time, it automatically moves to another day and time, not to the selected one.

**Comment by KsyushaM** [comment]
Hi @wgutierrezr,
There is a similar [ticket](https://www.devexpress.com/Support/Center/Question/Details/T627821/scheduler-appointments-are-incorrectly-positioned-at-the-second-interval-of-the-workweek) in our Support Center. We have fixed this problem and will include it in the upcoming release.

**Comment by dxbykov** [comment]
Thanks to everybody who participated in this discussion or gave us early feedback by other means. The feature is now available in the [v18.1](https://github.com/DevExpress/DevExtreme/releases/tag/18.1.3) release. I'm closing this thread. In the case of bugs or questions, feel free to create new GitHub [issues](https://github.com/DevExpress/DevExtreme/issues) or tickets in our [Support Center](https://www.devexpress.com/Support/Center/).
